Tmain/omit-long-patterns-etags.d/stdout-expected.txt

```diff
@@ -0,0 +1,5 @@
+
+input.sh,349
+                                                                                                func96(func961,0
```

Comment:
I though truncating the head wasn't an option because etags matching used an anchored search? (`^...`)